### DISCUSSION TITLE
*.image property for my own postgresql image

### DISCUSSION THREAD

**Discussion by michael-yerkin** [testcontainers/testcontainers-java#4553] [General]
Hi!
Which name *.image property for my own postgresql image i have to use in testcontainers.properties?

**Comment by kiview** [comment]
If you use `PostgresqlContainer`, just specify it directly in the constructor. For substituting an image when the JDBC proxy is used, see: 
https://www.testcontainers.org/features/image_name_substitution/#developing-a-custom-function-for-transforming-image-names-on-the-fly
